libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Configuration, DrivingLicenseBookApi } from '../../gen/fetch'
+
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import { XRoadConfig } from '@island.is/nest/config'
+import type { ConfigType } from '@island.is/nest/config'
+import { DrivingLicenseBookClientConfig } from './drivingLicenseBookClient.config'
+import { Injectable, Inject } from '@nestjs/common'
+import { AuthHeaderMiddleware } from '@island.is/auth-nest-tools'
+
+@Injectable()
+export class DrivingLicenseBookClientService {
```

Comment:
I would consider to rename this class to `DrivingLicenseBookClientFactory` as it is more of a factory creating a new instance of the api with auth setup. The `api()` could then be renamed to `create()`

Reply:
Done.